README.rst

```diff
@@ -87,9 +87,56 @@ Requirements
 ------------
 SageMaker TensorFlow extensions builds on Python 3.4-3.8 in Linux with a TensorFlow version >= 1.7. Older versions of TensorFlow are not supported. Please make sure to checkout the branch of sagemaker-tensorflow-extensions that matches your TensorFlow version.
 
-Please note that: 
-TensorFlow extensions 2.2.0 only builds on Python 3.7.
-TensorFlow extensions 2.6.0 only builds on Python 3.8.
+Please refer to below table for release support information:
+
+.. list-table:: Sagemaker TensorFlow Extensions Release Information
+   :widths: 25 25 25 25
+   :header-rows: 1
+
+   * - Sagemaker TensorFlow Extensions Pypi Version
```

Comment:
Please change Pypi to PyPi

Reply:
I think we can ignore this nit for now. We need to merge and release asap.